src/Makefile.bench.include

```diff
@@ -43,6 +43,10 @@ bench_bench_bitcoin_LDADD = \
   $(LIBBITCOIN_UTIL) \
   $(LIBBITCOIN_CONSENSUS) \
   $(LIBBITCOIN_CRYPTO) \
+  $(LIBBITCOIN_WALLET) \
+  $(LIBBITCOIN_SERVER) \
+  $(EVENT_PTHREADS_LIBS) \
```

Comment:
Could you explain these? Compiles successfully with just the EVENT_LIBS on my machine.